lib/es/management/commands/reindex.py

```diff
@@ -117,6 +117,9 @@ class Command(BaseCommand):
         make_option('--with-stats', action='store_true',
                     help=('Whether to also reindex AMO stats. Default: False'),
                     default=False),
+        make_option('--quiet', action='store_true',
```

Comment:
I'd suggest renaming `quiet` to `noconfirm` as that's a bit more direct. `quiet` sounds more like it would suppress output which isn't strictly true.